utils/keys_test.go

```diff
@@ -252,17 +274,70 @@ func TestImportNoPath(t *testing.T) {
 	b := &pem.Block{
 		Headers: make(map[string]string),
 	}
-	b.Bytes = make([]byte, 1000)
-	rand.Read(b.Bytes)
+	from, _ := os.OpenFile("../fixtures/secure.example.com.key", os.O_RDONLY, notary.PrivKeyPerms)
+	defer from.Close()
+	b.Bytes, _ = ioutil.ReadAll(from)
 
-	bBytes := pem.EncodeToMemory(b)
+	in := bytes.NewBuffer(b.Bytes)
+
+	err := ImportKeys(in, []Importer{s}, "root", "", passphraseRetriever)
+	require.NoError(t, err)
+
+	for key := range s.data {
+		// no path but role included should work
+		require.Equal(t, key, filepath.Join(notary.RootKeysSubdir, "12ba0e0a8e05e177bc2c3489bdb6d28836879469f078e68a4812fc8a2d521497"))
+	}
+
+	err = ImportKeys(in, []Importer{s}, "", "", passphraseRetriever)
```

Comment:
Can we reset `s =NewTestImportStore()` here before importing again?  That way we can assert that there is no data in `s`?  You are totally correct, and that the new import should fail, but but because it's importing the same key again just without a role and gun, we can't distinguish whether it actually failed to import or imported it again to the same name (which shouldn't happen, because it cannot have formulated a name, but just in case :))